dragonfly/_base.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+"""Base class for all geometry objects."""
+from honeybee.typing import valid_string
+
+
+class _Base(object):
```

Comment:
Ok. In that case, do you think we should do the same in honeybee-core with [_Base](https://github.com/ladybug-tools/honeybee-core/blob/master/honeybee/_base.py#L6) and [_BaseWithShade](https://github.com/ladybug-tools/honeybee-core/blob/master/honeybee/_basewithshade.py#L7)?

Reply:
Do we really need to do that here? It depends on how much detail Dragonfly objects can take.

Reply:
I don't understand what you mean. Weren't you the one saying that we should be changing the object names to `_BaseGeometry` and `_BaseGeometryWithShade`? Now you are asking if we really need to change them? 
Personally, I am fine with either `_Base` or `_BaseGeometry` as long as it's consistent between Dragonfly and Honeybee. Just let my know what you are suggesting.

Reply:
My first comment was about changing the name of `_Base` to `_BaseGeometry` which I think is more appropriate.

The second comment was about  `_BaseGeometryWithShade`. Are we going to handle this much complexity in Dragonfly?

Reply:
No we aren't going to have a need for a `_BaseWithShade` in Dragonfly and I guess I originally had an opinion that I didn't want to make dragonfly-core inconsistent with honeybee-core conventions but now I don't have a strong opinion on the matter. I'll jut make it `_BaseGeometry` since it seems you have a stronger opinion on it than I.